### ISSUE TITLE
It is not possible to include a manual selection.

### ISSUE THREAD

**Issue by hericjr** [ihslimn/jet-form-builder-update-field#15]
I need this functionality but it must be added manually.

Ex: My selection field '_rooms' has the options: Room 1, Room 2 and Room 3.

In the selection field '_people' I want it to change according to what was selected in the field '_rooms', e.g.: if Room 1 is selected, the option to choose 1 Person or 2 people will appear in the options of the field '_people'. If Room 3 is selected, the option to choose 1 Person, 2 People or 3 People will appear in the options of the field '_people' and so on.

It would be ideal to add this option to this addon to insert the changes manually without depending on the query builder.

Basically, it is to insert a conditional to show certain selection options depending on what was selected in another field.

Thank you in advance.

**Comment by ihslimn** [comment]
This can be implemented either with custom field options generator, or rather with JS. 
It will definitely not be inplemented in this addon.
